lib/ts/scriptsUtils.ts

```diff
@@ -0,0 +1,76 @@
+import { Answers, UIBuildType } from "./types.js";
```

Comment:
why is this file called scriptsUtils if it's not really running an external script?